src/lib/io/sol-iio.c

```diff
@@ -43,6 +43,7 @@
 #endif
 
 struct sol_iio_device {
+    double *mount_matrix;
```

Comment:
Thanks for your comments. Not all sensors support such mount_matrix sysfs node, By using point instead of array, we can malloc memory for these supported sensors only, to save some memory